SwiftUITodo/TaskListView.swift

```diff
@@ -31,6 +31,7 @@ struct TaskListView: View {
   }
 
   private func createTask() {
+    print("task created")
```

Comment:
Can you get rid of this line?

Reply:
Oops, removed. just wanted to debug why I can't add a new task. It seems that the `onCommit` doesn't get called in Xcode 11 Beta 2. I haven't tested on real device though.